### PR TITLE
style: update EzCard styling for marketplace theme

### PR DESCRIPTION
**Pull request by danidewitt**
## What did we change?
- Updated styling for EzCard marketplace theme

## Why are we doing this?
- Request from @ktlnux 
Screen Shot 2022-04-20 at 4 28 16 PM

## Screenshot(s) / Gif(s):
Before:
Screen Shot 2022-04-20 at 4 44 07 PM

After:
Screen Shot 2022-04-20 at 4 44 21 PM

## Checklist

- [ ] Provide test coverage for the changes made
- [x] Create a changeset (`yarn changeset`) with a summary of the changes

[Contributing guidelines](https://ezcater.github.io/recipe/guides/contributing/)

### CODE REVIEW

**Review by pgrippi — APPROVED** [review]

**Comment by pgrippi** [comment]
I think we can close https://github.com/ezcater/recipe/issues/612 once this is merged!

File: packages/recipe-ui-theme-marketplace/src/index.ts [hunk 1]
```diff
@@ -2,15 +2,21 @@ import {Theme} from '@react-types/provider';
 import {createTheme} from '@ezcater/recipe';
 
 const marketplace = createTheme({
+  borderStyles: {
+    'card-border': 'solid',
+  },
+  borderWidths: {
+    'card-border': '1px',
```
Comment (danidewitt) [inline packages/recipe-ui-theme-marketplace/src/index.ts:9]: should change `1px` to `$thin`

**Review by pgrippi — APPROVED** [review]

**Review by noranda — DISMISSED** [review]
Looks great! All tests pass.

**Review by noranda — APPROVED** [review]

**Comment by noranda** [comment]
All tests pass. Not sure if this is an issue, but I did notice when switching between themes that they no longer line up exactly, one will shift slightly which may have something to do with border size differences.

**Review by pgrippi — DISMISSED** [review]

**Review by noranda — DISMISSED** [review]

**Comment by danidewitt** [comment]
@noranda @pgrippi in the last two commits, i updated all the places we were using placeholder.com since it's consistently having issues with database/hosting errors, and we use that service in tests, docs, etc. and i'm not super comfortable merging a PR without updating baseline visual regression snapshots. the only differences in the snapshots should be the font of the image itself, since i'm now using a different service.

**Review by pgrippi — APPROVED** [review]
the changes and snapshots look good to me! 🤞🏻 this service is a little more stable than placeholder.com!

**Review by pgrippi — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -10,5 +10,4 @@ packages/**/dist
 reports
 .eslintcache
 *.cache
-__diff_output__
```
Comment (pgrippi) [inline .gitignore:13]: was this removed intentionally? without this here you will run the risk of someone committing the diffs to source control if it one of the integration tests fails locally and they do a blanket `git add .`

**Review by noranda — APPROVED** [review]
All tests pass!

**Pull request merged**